src/Components/NewPostForm/NewPostForm.tsx

```diff
@@ -55,9 +54,9 @@ class NewPostForm extends Component<IProps, IPost> {
           value={this.state.content}
           onChange={event => this.handleChange(event)}
         />
-        <Link to={'/'} style={{ textDecoration: 'none' }}>
+        <Link to='/'>
           <button onClick={event => this.submitPost(event)}>Share</button>
-        </Link >
+        </Link>
```

Comment:
I don't think you need to pass the event into this method. You'll need to add event as a parameter in submitPost and then do event.preventDefault so that it won't reload. Then it needs a redirect.

Reply:
I think the content input can be changed to a textarea element so that a user can see their whole post.

Reply:
We actually had to take the event.preventDefault out because it prevents the link from going back to the home page, so event does not need to be passed in here. 
Textarea has been changed!